bdk-ffi/src/types.rs

```diff
@@ -592,3 +592,7 @@ impl From<BdkTx> for Tx {
         }
     }
 }
+pub struct UnconfirmedTx {
```

Comment:
If we go with the proc macros proposed above, a little bit of API docs to describe this type might be nice.

Reply:
Maybe something like this?

```rust
This type replaces the Rust tuple `(tx, last_seen)` used in the Wallet::apply_unconfirmed_txs` method, where `last_seen` is the timestamp of when the transaction `tx` was last seen in the mempool.
```

Reply:
perfect, updated